frontend/src/pages/RoomPage.tsx

```diff
@@ -200,6 +225,24 @@ export const RoomPage = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [partnerInfo]);
 
+  /* Tabular Data */
+  const tableHeaders = [
+    "DATE",
+    "PROBLEM",
+    "RUNTIME",
+    "LANGUAGE",
```

Comment:
We probably don't need to show them the language since they would have chosen it before entering and can't change it.